manifests/machineconfigdaemon/cookie-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cookie-secret
+  namespace: {{.TargetNamespace}}
+type: Opaque
+data:
+  cookie-secret-seed: {{.GenerateProxyCookieSecret}}
```

Comment:
q: what does cookie-secret-seed mean?

Reply:
I'm not really sure, will change to just `cookie-secret`